open-metadata-implementation/governance-servers/data-engine-proxy-services/data-engine-proxy-services-server/src/main/java/org/odpi/openmetadata/governanceservers/dataengineproxy/processor/DataEngineProxyService.java

```diff
@@ -150,12 +150,12 @@ public void run() {
 
     }
 
-    public void load() {
-        final String methodName = "load";
+    public void initialLoad() {
```

Comment:
I think that this is not necessarily an initial load, but a load done each time the endpoint it called. As per our discussion, it could also be an update done because an operator knows there are some changes in the data already loaded.

Reply:
Thinking about this twice I agree, there is no need to rename it, load is more general purpose; can be used for first time load or subsequent reload - no difference from proxy perspective it is always the same process and it is up to the consumer of events (data engine omas) to decide on what it does creates/updates etc. Reverting name change.